packages/g/glibc/package.yml

```diff
@@ -1,6 +1,6 @@
 name       : glibc
```

Comment:
Is `packages/g/glibc/files/perf/0001-Open-up-the-haswell-subdirectory-to-be-usable-by-sup.patch` still needed?